panther_hardware_interfaces/src/roboteq_driver.cpp

```diff
@@ -0,0 +1,372 @@
+// Copyright 2023 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <panther_hardware_interfaces/roboteq_driver.hpp>
+
+#include <chrono>
+#include <cmath>
+#include <cstdint>
+#include <future>
+#include <memory>
+#include <mutex>
+#include <stdexcept>
+#include <string>
```

Comment:
```suggestion
#include <chrono>
#include <cmath>
#include <condition_variable>
#include <cstdint>
#include <memory>
#include <mutex>
#include <stdexcept>
#include <string>
#include <system_error>
```
added condition_variable and system_error, removed future. Is future used, I haven't found it anywhere?

Reply:
I think that used it before, but now it should be removed